vagrant/k8s/Vagrantfile

```diff
@@ -98,6 +98,18 @@ SCRIPT
 cluster = read_cluster_config
 create_etc_hosts(cluster)
 
+def customize(v)
+  # make all nics 'virtio' to take benefit of builtin vlan tag
+  # support, which otherwise needs to be enabled in Intel drivers,
+  # which are used by default by virtualbox
+  # changes the settings for the first 5 NICs, regardless of presence
+  1.upto(5) do |n|
```

Comment:
@DivyaVavili: This doesn't change the order. That's not possible as far as I know. They're indexed and all the changes you run affects a particular one (e.g.: changing the setting for NIC with index 1 won't make it NIC with index 2; it won't affect the listing in the OS either).

I've made those changes to make the code future proof. Should we need more NICs, we don't need to make any changes for the next NICs we need.

k8s-l3-cluster is most likely broken if it hasn't been tested with the previous set of changes.